CMakeLists.txt

```diff
@@ -3,4 +3,16 @@ enable_testing()
 
 project(opentracing-cpp)
 
+set(OPENTRACING_ABI_VERSION "1")
```

Comment:
Yes, cmake commands aren't case sensitive. I was also considering maybe setting the abi version from the major and minor versions.